PythonTasks/nutritionist/nutritionist.py

```diff
@@ -0,0 +1,74 @@
+
+def height_in_meters(feet, inches):
+    f = float(feet)*0.305
+    i = float(inches)*0.0254
+    meters = f+i
+    return meters
+
+
+def weight_in_kgs(weight):
+    w = float(weight)*0.45
+    return w
+
+
+def bmi_calculation( w, meters):
+    bmi = w / (meters * meters)
+    print("Your BMI is: ",round(bmi,2)," kg/m^2")
+
+
+def bmr_calculations(gender, w, hcm, age):
+
+    if gender == "female":
+        bmr=(10*float(w))+(6.25*float(hcm))-(5*float(age))-161
+    else:
+        bmr = (10*float(w))+(6.25*float(hcm))-(5*float(age)+5)
+    return bmr
+
+
+def lean_body_mass(w , age, hcm, gender):
+    if age < "14":
+        ecm = 0.0125*(w**0.6469)*hcm-0.7236
+        return ecm
+    elif age > "14":
+
+        if gender == "female":
+            elbmf = 0.29569*w+0.42*hcm-43.2933
+            return elbmf
+        else:
+            elbmm = 0.32810*w+0.33939*hcm-29.5336
+            return elbmm
+
+
+name = input('Enter your name: ')
```

Comment:
how about encapsulating the main logic in a if code block. Kindly search how to write main function in python.